scripts/analysis_1/nbconverted/3_visualize_data.py

```diff
@@ -180,16 +180,10 @@
 # Add label
 input_data_UMAPencoded_df['color_by'] = normalized_data_label['color_by']
 
-ggplot(input_data_UMAPencoded_df, aes(x='1',y='2'))     + geom_point(aes(color='color_by'), alpha=1)     + xlim(7, 11)     + ylim(-3, -2.5)     + ggtitle('Input data')
-
-
-# In[28]:
-
-
-label_ls = normalized_data_label['color_by'].unique()
+ggplot(input_data_UMAPencoded_df, aes(x='1',y='2'))     + geom_point(aes(color='color_by'), alpha=1)     + xlim(7, 11)     + ylim(-3, -2.5)     + xlab('UMAP 1')     + ylab('UMAP 2')     + ggtitle('Input data')
```

Comment:
Need muted colors for `not in experiment` data points. e.g. grey

Reply:
I completely agree.  I have tried to play around with manually setting the colors but can't seem to find a way to do this though I'm sure there exists a solution

Reply:
matplotlib doesn't play nicely with automatic string to color assignments but you can use a dictionary. For smaller sets (4 in your case), hand-picking colors is better anyway.